### PR TITLE
[mod] get user and added aboutMe to the schema

### PR DESCRIPTION
**Pull request by luke-dowling**

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const { Schema, model } = mongoose;
 const UserSchema = new Schema(
   {
     username: { type: String, required: true },
+    aboutMe: String,
```
Comment (chmette) [inline models/User.js:7]: we did not set a required false and so we need to make it unique?

Comment (luke-dowling) [reply models/User.js:7]: I think required false is the default and why unique?

Comment (chmette) [reply models/User.js:7]: ah ok. well that you cant copy the description of someone else? its picky but..

**Review by chmette — APPROVED** [review]
thanks for the integration

**Pull request merged**